ckanext/canada/templates/user/perform_reset.html

```diff
@@ -0,0 +1,44 @@
+{% ckan_extends %}
+
+{% block primary_content %}
+  {% if g.is_registry %}
+    {# Registry #}
+    <div class="row">
+      <div class="col-md-12">
+        <section class="alert alert-info">
+          <p>
+            {{ h.render_markdown(_(
+                '1. Passwords must contain characters from at least 3 of the following 4 classes:\n'
+                '    * Upper Case Letters: A, B, C, … Z\n'
+                '    * Lower Case Letters: a, b, c, … z\n'
+                '    * Westernised Arabic Numerals: 0, 1, 2, … 9\n'
+                '    * Special characters: ({}[],.<>;:\'"?/|\\`~!@#$%%^&*()_-+=).\n'
+                '2. No accented characters are permitted.\n'
+                '3. All passwords must be at least 8 characters long.\n')) }}
```

Comment:
One day we implement the modern password standards I hope...

Reply:
I think there was some requirements stated for having the Registry go public. But I think the requirements that were given are just these ones. :/

Reply:
Check out the new password complexity standards from the link in the ticket @RabiaSajjad showed this morning. We don't need all this with the new standards.

Reply:
Yeah, the new standards will probably be done separately from the template merge. We will update the text here then.